src/main.js

```diff
@@ -60,23 +60,63 @@ console.log(min(6,4));
     return y;
 }
  */
-function max(a,y) {
-    return a > y ? a : y;
+function max(a, y) {
+  return a > y ? a : y;
 }
-console.log(max(4,8));
-console.log(max(10,2));
+console.log(max(4, 8));
+console.log(max(10, 2));
 
-let randomArray = [];
+const randomArray = [];
 for (let i = 0; i < 10; i++) {
-    randomArray.push(Math.round(Math.random() * 100));
+  randomArray.push(Math.round(Math.random() * 100));
 }
 console.log(randomArray);
 
 function replace(number) {
-    if (`${number}`.includes("0")) {
-        return `${number}`.replaceAll("0", "zero");
-    } else {
-        return number;
-    }
+  if (`${number}`.includes('0')) {
+    return `${number}`.replaceAll('0', 'zero');
+  } else {
+    return number;
+  }
 }
 console.log(randomArray.map(replace));
+
+function sum(a) {
```

Comment:
It's becoming too much code in one file. Please, do your next homework in a separate file. You can imoport scripts in main.js